### ISSUE TITLE
listen for hit event in addition to kill event to check HP for soft kills

### ISSUE THREAD

**Issue by DanAlbert** [dcs-liberation/dcs_liberation#3313]
https://wiki.hoggitworld.com/view/DCS_func_getLife and https://wiki.hoggitworld.com/view/DCS_func_getLife0 can be used to determine how damaged a unit is. We could listen for https://wiki.hoggitworld.com/view/DCS_event_kill and check the health, and register the kill after some predefined amount (settings controllable, probably) rather than requiring a total kill. It'd make rockets and most CBUs less useless.

Within a single mission a soft kill is often enough to render the unit inoperable anyway, there's just no way for us to carry the damage over between turns. This would sort of approximate that.

**Comment by SnappyComebacks** [comment]
Hopefully there's other options, but the two possible options I could suggest to simulate a soft kill is to force ROE green, or set the unit as static / unmanned.

**Comment by DanAlbert** [comment]
DCS AI won't know that those targets don't need to be dealt with again. Better to just kill them.

**Comment by Starfire13** [comment]
Yeah. Would be very nice if parked aircraft would blow up after getting strafed, instead of just smoking till they despawn. Would mean no longer having to use 2000lb bombs on a plane to ensure a kill. 20 mike mike, rockets, etc would become useful.
